subprojects/zstd.wrap

```diff
@@ -8,4 +8,4 @@ patch_filename = zstd-1.4.5-1-wrap.zip
 patch_hash = fd9cb7b9c8f7092ef1597ff68f170beef65fcf33e575a621955cf405a41db1cc
 
 [provide]
-libzstd = libzstd_dep
+zstd = zstd_dep
```

Comment:
This wrap file is correct when downloaded from the official WrapDB repository, and needs to align with what is actually exported by the meson.build files in the zstd project.

Reply:
Yep, I'd just figured that out and have got `zstd` working properly as a subproject.  I think I'm close to making `curl` an optional dependency (where `zck_dl` doesn't build without it), which should then allow us to build this without mamba.